memcache/memcache_test.go

```diff
@@ -212,6 +212,169 @@ func testWithClient(t *testing.T, c *Client) {
 	// Test Ping
 	err = c.Ping()
 	checkErr(err, "error ping: %s", err)
+
+	//test Meta get command
+	testMetaGetWithClient(t, c, checkErr)
+}
+
+func testMetaGetWithClient(t *testing.T, c *Client,
+	checkErr func(err error, format string, args ...interface{})) {
+	c.DeleteAll()
+	defer c.DeleteAll()
+
+	key := &Item{Key: "key", Value: []byte("value")}
+	err := c.Set(key)
+	checkErr(err, "first set(key): %v", err)
```

Comment:
I believe check err is used to identify where the error has occurred instead of explaining about the test case. I have added some comments on the top to address what I am doing.